<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>857</width>
    <height>517</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Антиплагиат-текст</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>841</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Введите текст для проверки</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditIn"/>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonCompare">
        <property name="text">
         <string>Сравнить</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="pushButtonAdd">
        <property name="text">
         <string>Добавить в БД</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Сранительные характеристики</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditResInfo">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Самый похожий текст из БД</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEditOut">
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
